omero/sysadmins/version-requirements.rst

```diff
@@ -393,6 +427,8 @@ Package lists
       - `Reference <http://mirror.centos.org/centos/6/os/x86_64/Packages/>`__
     * - CentOS 7 / RHEL 7
       - `Reference <http://mirror.centos.org/centos/7/os/x86_64/Packages/>`__
+    * - CentOS 8 / RHEL 8
+      - `Reference <http://mirror.centos.org/centos/8/os/x86_64/Packages/>`__
```

Comment:
FYI it'll be a while before CentOS8 is released https://wiki.centos.org/About/Building_8